src/Sylius/Bundle/AdminBundle/Menu/MainMenuBuilder.php

```diff
@@ -101,4 +102,21 @@ private function configureMarketingSubMenu(ItemInterface $menu)
             ->setLabelAttribute('icon', 'in cart')
         ;
     }
+
+    /**
+     * @param ItemInterface $menu
+     */
+    private function configureAssortmentSubMenu(ItemInterface $menu)
+    {
+        $assortmentSubMenu = $menu
```

Comment:
Why variable and function is named "assortment menu" if we add "sylius.menu.admin.main.**catalog**.header"? :)